Userland/Libraries/LibGfx/Font/OpenType/Tables.h

```diff
@@ -42,6 +43,25 @@ using Tag = BigEndian<u32>;
 using Offset16 = BigEndian<u16>;
 using Offset32 = BigEndian<u32>;
 
+// ISO-IEC 14496-22:2019 4.5.1 Offset table
```

Comment:
Would it be better to use links to MSDN instead? I notice that's what most of this file does, but I wasn't sure if that's an "official" spec or just an MS interpretation/explanation.

Reply:
Decided to go with the MS ones.